ui/lib/utils/sqlFormatter/index.ts

```diff
@@ -1,7 +1,13 @@
-import TiDBSQLFormatter from './TiDBSQLFormatter'
+// import TiDBSQLFormatter from './TiDBSQLFormatter'
 
-const mySqlFormatter = new TiDBSQLFormatter({ uppercase: true })
+// const mySqlFormatter = new TiDBSQLFormatter({ uppercase: true })
+
+// export default function formatSql(sql?: string): string {
+//   return mySqlFormatter.format(sql || '')
+// }
+
+import { format } from 'sql-formatter'
```

Comment:
It will crash when using the forked formatter, and the upstream still keep updated.

![image](https://user-images.githubusercontent.com/1284531/150890431-f13827fb-d2bd-4bec-9bd9-3511bdfe641a.png)

Reply:
But as we are specially introducing a TiDB formatter, will there be any feature breaks when it is not used?

Reply:
I did some manual tests, it looks the same as before, but I'm not 100% confident, I think we should add some unit tests for this method, I will book a todo.